docs/data-sources/equinix_fabric_network.md

```diff
@@ -0,0 +1,99 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "equinix_fabric_network Data Source - terraform-provider-equinix"
+subcategory: ""
```

Comment:
Add subcategory "Fabric" so that it lands in the right section on the registry page.